pkg/pillar/cmd/vaultmgr/vaultmgrzfs.go

```diff
@@ -86,6 +86,17 @@ func createZfsVault(vaultPath string) error {
 	return nil
 }
 
+//remove vaults from zfs
```

Comment:
See yetus complaint below. (yetus might be confused about which line is new but this line doesn't follow the whitespace convention and is new)